packages/php.rb

```diff
@@ -28,7 +28,7 @@ def self.preflight
     puts '8.2 = PHP 8.2.1'
     puts '  0 = Cancel'
 
-    while version = $stdin.gets.chomp
+    while version = $stdin.gets.chomp.downcase
```

Comment:
No need for `.downcase` here since all the expected input will be numeric.

Reply:
Ah well, you can fix it on the next update. :)